spec/factories/users.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    phone_number { Faker::PhoneNumber.cell_phone }
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
+    password { 'password' }
+    password_confirmation { 'password' }
+    address { association :address, user: instance }
+
+    trait :as_help_seeker do
+      role { 0 }
```

Comment:
Or even use `User.roles[:help_seeker]`, this way we don't need to know the exact values, just the key.